### ISSUE TITLE
Text in a different language/direction

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#453]
**Is your feature request related to a problem? Please describe.**
Text in a different language/direction to other text, or the language/direction of the page, should have a specific treatment so screen readers can read out the text properly.

**Describe the solution you'd like**
The text that has the different language/direction should be wrapped in a span element. This span element should have a `dir` of the required direction, and a `lang` of the required language.

- Ensure we have examples of multiple languages/text directions in a fixture data file & link to it in the PR description (This will be with a fragment block with attribute `lang`) (here's a link to an ares article on Test with News/English/LTR at the top-level and Farsi/RTL in a fragment. 
- Ensure we have an example of this working in Storybook - could live in Paragraph or Text or Fragment. To be decided when worked on. 
- When testing, check Storybook and also the fixture data test article. 

**Testing notes**
- When visiting a News article, canonical or amp, with a text fragment with the Farsi language, you should see the attribute `lang="fa"`.

**Additional Context**
~This is dependent on knowing what the nested text will look in the passed data.~ we now know the structure. 
@jimjohnsonrollings

**Comment by sareh** [comment]
If this is the whole headline/subheading component, then there should not be an additional `<span>`, but the attributes mentionted (`lang` & `dir`) should be applied to the `<h1>` or `<h2>`.

**Comment by katikaa** [comment]
This is going to be used in those components: text block, headline or subheading, image caption, image copyright, image alt-text.
Reasons for moving it to 1.1:
- the main use case for this is going to appear in WS more frequently
- equivalent Optimo requirement is phase 1

**Comment by BogdanDogaru** [comment]
Blocked pending work in our CMS to expose the language of text fragments in a paragraph.

**Comment by sareh** [comment]
Now unblocked, since the data structure is clarified - there will be fragments with `lang` attributes.

**Comment by sareh** [comment]
Blocked due to Optimo not surfacing the lang attributes for fragment blocks yet. This issue will be unblocked as soon as Optimo enables multiple languages.
